### DISCUSSION TITLE
Correct way to access command options

### DISCUSSION THREAD

**Discussion by pswu11** [GrapesJS/grapesjs#5522] [Q&A] [answered]
Hello all, I have read the documentation about commands and try to understand how options work. Here is what I'm trying to do:

A dummy command that is supposed to accept options and add a section accordingly
```tsx
commands.add("add-section", {
  run(editor: Editor, options: {index: number, section: string}) {
    editor.getComponents().add(`${options.section}`,{ at: options.index })
  }
}) 
```

and then a button that will run this command:
```ts
<Button
    colorScheme="gray"
    onClick={async () => {
      const section = await createSection.mutateAsync(
        {content: EXAMPLE_TEXT}
      ) // here we received something similar to `<section>hi</section>` successfully
      commands.run("add-section", {index: 4, section: section})
    }}
>
  Prompt
</Button>
```

The command is executed when the button is clicked but for some reason `options.index` and `options.section` are always `undefined`. Does anyone know where I might have overlooked? Thank you in advance! 🙏🏼

**Comment by artf** [comment, marked as answer]
<img width="851" alt="Screenshot 2023-11-21 at 01 37 29" src="https://github.com/GrapesJS/grapesjs/assets/11614725/75a3df60-e31c-4f8b-bced-a03202fabcf2">

**Reply by pswu11** [reply]
Oops 😂 thanks a lot!
